embedded-bins/Makefile

```diff
@@ -2,7 +2,7 @@
 runc_version = 1.0.0-rc92
 containerd_version = 1.4.3
 kubernetes_version = 1.19.4
-kine_version = 0.5.1
+kine_version = 0.6.0
```

Comment:
This needs to be rebase due to https://github.com/k0sproject/k0s/pull/480

Reply:
Done